crates/subspace-networking/src/bin/subspace-bootstrap-node/main.rs

```diff
@@ -220,3 +229,55 @@ async fn main() -> Result<(), Box<dyn Error>> {
 fn peer_id_from_keypair(keypair: Keypair) -> PeerId {
     peer_id(&libp2p::identity::Keypair::from(keypair))
 }
+
+/// Networking manager implementation for DSN bootstrap node.
+#[derive(Clone, Default)]
+pub(crate) struct NetworkingParametersManager {
```

Comment:
It is confusing that we have a second `NetworkingParametersManager` here that does different things. It should be named differently. Probably both structs (and trait) should have a name closer to `KnownPeers` (which is the file the other one created).